platform/android/export/export_plugin.cpp

```diff
@@ -2225,13 +2225,6 @@ String EditorExportPlatformAndroid::get_apksigner_path(int p_target_sdk, bool p_
 }
 
 bool EditorExportPlatformAndroid::has_valid_export_configuration(const Ref<EditorExportPreset> &p_preset, String &r_error, bool &r_missing_templates, bool p_debug) const {
-#ifdef MODULE_MONO_ENABLED
-	// Don't check for additional errors, as this particular error cannot be resolved.
-	r_error += TTR("Exporting to Android is currently not supported in Godot 4 when using C#/.NET. Use Godot 3 to target Android with C#/Mono instead.") + "\n";
-	r_error += TTR("If this project does not use C#, use a non-C# editor build to export the project.") + "\n";
-	return false;
-#else
```

Comment:
If the Android export is still a work in progress, it may be better to keep a message as a warning (without setting valid to `false`). Something like:

```C++
#ifdef MODULE_MONO_ENABLED
	err += TTR("Exporting to Android when using C#/.NET is experimental.") + "\n";
#endif
```

Reply:
Indeed, but I need to familiarise myself with github first.

Reply:
> If the Android export is still a work in progress, it may be better to keep a message as a warning (without setting valid to `false`). Something like:如果 Android 导出仍在进行中，最好保留一条消息作为警告（不将 valid 设置为 `false` ）。就像是：
> 
> ```c++
> #ifdef MODULE_MONO_ENABLED
> 	err += TTR("Exporting to Android when using C#/.NET is experimental.") + "\n";
> #endif
> ```
How does it look?
![image](https://github.com/godotengine/godot/assets/52148221/3ebc3c77-7604-4afe-a6e8-d2b6793b69e1)

Reply:
Looks good to me.